lib/anonymizer.js

```diff
@@ -0,0 +1,571 @@
+var http = require('http'),
+    https = require('https'),
+    url = require('url'),
+    util = require('util'),
+    cluster = require('cluster'),
+    numCPUs = require('os').cpus().length,
+    events = require('events'),
+    exec = require('child_process').exec,
+    gh = require('./lib-helper.js'),
+    proxy = require('./ThrottleProxy/throttleServer.js'),
```

Comment:
Why do you need ThrottleProxy in anonymizer?

Reply:
I accidentally replaced the code of anonymizer with main.js, I am recovering it now